client/my-sites/plans/current-plan/jetpack-checklist/constants.ts

```diff
@@ -65,7 +65,17 @@ export const JETPACK_SECURITY_CHECKLIST_TASKS: ChecklistTasks = {
 };
 
 export const JETPACK_PERFORMANCE_CHECKLIST_TASKS: ChecklistTasks = {
-	// No tasks for this checklist yet…
+	jetpack_lazy_images: {
+		title: translate( 'Lazy Load Images' ),
```

Comment:
Hi! I've found a possible matching string that has already been translated **20** times:
`translate( 'Lazy load images' )` *ES Score: 11*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

Reply:
@eeeeevon13 / @jeffgolenski Thoughts on casing and translation re-use?